src/handle/mod.rs

```diff
@@ -29,7 +29,7 @@ pub trait Managed: Copy {
 
 /// A safely rooted _handle_ to a JS value in memory that is managed by the garbage collector.
 #[repr(C)]
-#[derive(Clone, Copy)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
This is a good change (https://rust-lang.github.io/api-guidelines/interoperability.html#c-common-traits), but my preference is to minimize unrelated changes in a PR.

Reply:
This was needed to call `date_underflow.unwrap_err` in tests